### PR TITLE
Fix a few typos in README, docs and docstrings

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

**Review by tomchor — APPROVED** [review]

**Review by francispoulin — APPROVED** [review]

File: docs/src/appendix/benchmarks.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ Environment:
 └─────────────┴─────┴─────────┴─────────┴─────────┘
 ```
 
-Like the shallow water model, it can be seen at grid size `64^3` that the GPU is beginning to be saturated as speedups rapidly increase. At grid sizes `128^3` and `256^3` we see the speedup stablise to around 400 times.
+Like the shallow water model, it can be seen at grid size `64^3` that the GPU is beginning to be saturated as speedups rapidly increase. At grid sizes `128^3` and `256^3` we see the speedup stabilise to around 400 times.
```
Comment (glwagner) [inline docs/src/appendix/benchmarks.md:146]: Should be "stabilize"!

**Comment by glwagner** [comment]
> No description provided.

Thanks!

**Comment by pitmonticone** [comment]
> Thanks!

Thank you all for the development and maintenance of this wonderful Julia package!

File: docs/src/appendix/benchmarks.md (hunk 1 above)
Comment (pitmonticone) [reply docs/src/appendix/benchmarks.md:146]: Ops I'm sorry. I've assumed you wanted to adopt the British version.

**Pull request merged**